docs/source/introduction.rst

```diff
@@ -0,0 +1,50 @@
+Introduction
+============
+
+Nextcloud-API is Python (2 and 3) wrapper for NextCloud's API. With it you can manage your
+NextCloud instances from Python scripts.
+
+If you have any question, remark or if you find a bug, don't hesitate to
+`open an issue <https://github.com/EnterpriseyIntranet/nextcloud-API/issues>`_.
+
+
+
+
+Quick start
+-----------
+
+First, create your NextCloud instance:
+
+.. code-block:: python
+
+    from NextCloud import NextCloud
+
+    nxc = NextCloud.NextCloud("url", "user id", "password", js=True)
+
+Then you can work with NextCloud objects:
+
+.. code-block:: python
```

Comment:
This looks good so far, if you want to improve it, consider making the documentation based on tests - now, you have manually written some example code, but if you would include code from a test, it would guarantee that the code is always up-to-date, performing as expected. The sphinx [literalinclude](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude) directive has huge possibilities in this regard, for example `start-after` and `end-before`.

Reply:
I also added code examples to `example.py` file and linked them in separate Examples page in documentation. Should I delete that file and link all the examples from tests ? 
Thanks!

Reply:
My point was that it may happen that the documentation falls out of sync with actual state, if it is not tested. Extracting the code into a separate file is surely a step forward, but if the example is not tested, then it is still not "there".
However, I feel that we are quite close - if you insert some asserts there, for example create a user, then assert that it exist when you list all users, you remove it, it doesn't exist etc., then you not only have a test for free, but users and readers exactly know what behavior to expect. And the only thing to figure out would be how to include that in an actual test.
But I think that sharing the file to a container and executing it using the python interpreter should do the trick - it just has to be documented in the test README.

Reply:
I understand. I added asserts to example.py code and linked it in Introduction as quick start examples and it looks good. 
Should we include examples of all methods like this in separate Examples page in documentation on the beginning of the project? Because it will take a lot of time and will be almost the same as copy-pasting tests but without self.nxc and asserts in between the code.

Reply:
Of course not, you are right - no copy pasting! I like changes to the example. Let's stick with it for the time being, more docs can always be added in another PR.